### PR TITLE
Update component “expanders”

### PR DESCRIPTION
**Pull request by itsmemin**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/components/expanders.md [hunk 1]
```diff
@@ -1,141 +1,196 @@
...
+We recommend that all Expanders be in the default state when the user navigates to the page. This ensures that as many sections as possible are visible to the user immediately. 
+
+By default, all sections can be opened at the same time. If there are any sections that contain actions, then consider limiting the user to opening one section at a time to keep the user focused. 
+
+### Keyboard navigation
+
+* Tab to select an Expander
+* Space bar/ enter key to open or close the selected section
+* When open, tab to select interactive elements (input, link or buttons if available)
+* Tab to select the next Expander
```
Comment (juliexiong) [inline src/_areas/design-system/components/expanders.md:119]:
```suggestion
* `[Tab]` to select an Expander
* `[Space]` or `[Enter]` to open/close the selected section
* When open, `[Tab]` to select interactive elements (input, link or buttons if available)
* `[Tab]` to select the next Expander
```
Just formatting to make it more obvious that these are keys that should be pressed on a keyboard.

File: src/_areas/design-system/components/expanders.md [hunk 2]
```diff
@@ -1,141 +1,196 @@
...
+- - -
 
-## Alternative considerations
+## Alternate considerations
 
-- Consider using **Cards** when you need to the ability to view and edit a group of information all at once. Show read only data on the card with an edit button that changes everything to be editable.
-- Consider using [**Tabs**](/design-system/components/tabs/) instead of Expanders when there is a lot of information in the content area of a section that would push other sections out of the user's viewport, or if there is a lot of complex information.
-- Consider using **Jump links** when it would be best to have all the content visible and the user just needs to jump to different sections on the page.
+* Consider using [Cards](/design-system/components/cards/) when there is no real benefit to focusing the user on one section at a time.
+* Consider using [Tabs](/design-system/components/tabs/) when the length of the content area exceeds the average users' viewport height or when there are more than a handful of form fields.
```
Comment (juliexiong) [inline src/_areas/design-system/components/expanders.md:196]:
```suggestion
* Consider using [Tabs](/design-system/components/tabs/) when the length of the content area exceeds the average user's viewport height or when there are more than a handful of form fields.
```

File: src/_areas/design-system/components/expanders.md [hunk 3]
```diff
@@ -1,141 +1,196 @@
...
+7. **Action area (optional):** An optional area for consistent placement for actions inside of a section.
 
-<Spacer size="large" />
+<docs-spacer size="small"></docs-spacer>
 
-<div id="expander--header-details"></div>
+### Peek-a-boo Expander
 
-### Adding header details
+A Peek-a-boo Expander contains 3 different parts.
```
[the remark this answers is not in the document]
Comment (itsmemin) [reply src/_areas/design-system/components/expanders.md]: Ok, so the initial preview lines of text are not even included in the component? 

I can update and add some clarity around that any titles or preview text it needs to be added before using the actual component itself. I think we'll want to have the title and preview text documented still as that is providing context around what you are going to "see more" of and not having at least some preview text feels a bit odd.

[the remark this answers is not in the document]
Comment (juliexiong) [reply src/_areas/design-system/components/expanders.md]: Yup, agreed that it's still good to document title and preview text. Calling out that they're not part of the peek-a-boo component is only what's missing.

**Comment by dave52** [comment]
This looks good IMO, I prepped a MR to ZUI updating demo's that follow the revised expander guidance and use, once that's in and deployed, the booster demo tab will sync up with what you have here.

File: src/_areas/design-system/components/expanders.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (itsmemin) [reply src/_areas/design-system/components/expanders.md]: @juliexiong Could you take one more look at the updates on this section. After that I think we are good to go!

[the remark this answers is not in the document]
Comment (juliexiong) [reply src/_areas/design-system/components/expanders.md]: Looks good! Merging in.

**Pull request merged**